js/components/forms/utils.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Useful methods to interact with redux-form
+ *
+ * @flow
+ */
+
+import * as React from 'react'
+import { type FieldProps } from 'redux-form'
+import { Item, Input, View, Text } from 'native-base'
+
+import I18n from '../../i18n'
+
+const getValidatorMessage = (validatorId: string): string => {
+  return I18n.t(`forms.validators.${validatorId}`)
+}
+
+const required = (value: string): ?string =>
+  value ? undefined : getValidatorMessage('required')
+
+const email = (value: string): ?string =>
+  value && !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value)
```

Comment:
maybe it's worth to use a validator library (ie. https://www.npmjs.com/package/validator)

Reply:
yes I agree